tests/aws/services/lambda_/functions/lambda_echo_json_body.py

```diff
@@ -18,8 +18,18 @@ def handler(event, context):
 
 import json
 
-
 def handler(event, context):
-    # Just print the event that was passed to the Lambda
-    print(json.dumps(event))
-    return json.loads(event["body"])
+    print(f"Received event: {json.dumps(event)}")
```

Comment:
Is there a reason for this `"Received event"` to lead the event log string?

The issue is that including this makes the logs invalid JSON, hence you'll see your `lambda-logs` snapshot is not transformed/snapshotted correctly.

I'd also expect this addition to break some other tests so rather remove this from the handler 💭 

```suggestion
    print(json.dumps(event))
```